spec/controllers/scim_rails/scim_groups_controller_spec.rb

```diff
@@ -158,7 +158,47 @@
         http_login(company)
       end
 
-      # TODO: add tests once method is implemented
+      let(:user_list_length) { 3 }
+      let(:group_name) { Faker::Games::Pokemon.name }
+      let(:invalid_id) { "invalid_id" }
+
+      let!(:user_list) { create_list(:user, user_list_length) } 
+      let!(:group) { create(:group, display_name: group_name, users: user_list, company: company) }
+
+      let(:returned_resource) { JSON.parse(response.body) }
+
+      it "returns scim+json content type" do
+        get :show, { id: 1 }
+
+        expect(response.content_type).to eq "application/scim+json"
```

Comment:
🌵 I don't know if we want to get this fancy for this gem, but we could make this a re-usable shared example or something.
Seems like we'd be checking this on every spec